TestCases/serial_regression.py

```diff
@@ -490,7 +490,7 @@ def main():
     centrifugal_stage.cfg_dir   = "turbomachinery/centrifugal_stage"
     centrifugal_stage.cfg_file  = "centrifugal_stage.cfg"
     centrifugal_stage.test_iter = 100
-    centrifugal_stage.test_vals = [-10.166364, 1.621172, 2.206476e+01, 5.271075e-01] #last 4 columns
+    centrifugal_stage.test_vals = [-10.166717, 1.620590, 30.003500, 0.487959] #last 4 columns
```

Comment:
Hi Heather, thanks a lot for this check because there was indeed a small bug related to the rotating frame! This should be fixed now and I also checked the final solution.